### DISCUSSION TITLE
Easiest way to tracking overall attendance - Scan out question

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#4439] [Q&A] [answered]
Hey guys,

Hope all is well. I am trying to figure out the easiest way to track and manage current attendance during the event. Event offers 200 weekend, 800 Saturday and 800 Sunday passes. Lets say each day attendee plus weekend attendee tickets sells out (ends up being total capacity of 1,000 per day).  In a scenario where the day of Saturday you are at 1,000 max capacity at once but eventually 100 weekend + 200 Saturday attendees eventually leave, then it would be beneficial for the event to open an additional 300 for the Saturday if those weekend attendees don't return. Problem is we don't know if they will return and it may cause an issue if they choose to return. Here is what I am looking to do

If we have the weekend and Saturday attendees scan out, we can setup it up and see that they scanned out and the quota opens in the backend. The question is; Are we able to setup a scan out question asking if the attendee is returning? This way we know that we can safely open more quota and not overcrowd the event to more than 1,000 at once. 

A question like "Will you return today? Yes or No. If they click no, the system opens quota. If they say yes, the quota of attendance stays the same.

Is this possible? If so how can it be done?

**Comment by raphaelm** [comment]
No, unfortunately such functionality does not exist.

**Reply by raphaelm** [reply, marked as answer]
A quick hack would be only scanning at exit if they do not want to return ;)

**Reply by jxs714** [reply]
> A quick hack would be only scanning at exit if they do not want to return ;)

Thank you! @raphaelm
